android/build.gradle

```diff
@@ -26,6 +26,6 @@ subprojects {
     project.evaluationDependsOn(':app')
 }
 
-task clean(type: Delete) {
+tasks.register("clean", Delete) {
```

Comment:
Did your environment made this change? This shouldn't be included in this PR

Reply:
Hi Miguel, I am pulling the most recent main now